arangod/Aql/TraversalExecutor.h

```diff
@@ -68,6 +68,11 @@ class TraversalExecutorInfos {
       std::unordered_map<TraversalExecutorInfosHelper::OutputName, RegisterId,
                          TraversalExecutorInfosHelper::OutputNameHash>
           registerMapping,
+      // TODO [GraphRefactor]: Performance Improvement
+      // Idea: To replace map type with a array of ids only []
+      // Otherwise "invalid id" store. Will be less expensive without the
+      // hashing mechanism we use right now.
+
```

Comment:
I think we should make ourselves a ticket for this. and remove the comment from the code

Reply:
https://arangodb.atlassian.net/browse/GORDO-1541

created.